Library/Formula/json-fortran.rb

```diff
@@ -0,0 +1,73 @@
+class JsonFortran < Formula
+  desc "A Fortran 2008 JSON API"
+  homepage "https://github.com/jacobwilliams/json-fortran"
+  url "https://github.com/jacobwilliams/json-fortran/archive/4.1.1.tar.gz"
+  sha256 "97f258d28536035ef70e9ead5c7053e654106760a12db2cc652587ed61b76124"
+
+  head "https://github.com/jacobwilliams/json-fortran.git"
+
+  option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
+  option "with-keg-only", "Keg only install (Skip linking json-fortran.) Required to support multiple Fortran compilers."
+  option "without-make-test", "Skip running build-time tests (not recommended)"
+
+  depends_on "cmake" => :build
+  depends_on "cmake"
+  depends_on "robodoc" => :recommended
+  depends_on :fortran
+
+  keg_only "User selected keg only install" if build.with? "keg-only"
+
+  def caveats; <<-EOS.undent
```

Comment:
:heavy_check_mark: Scrapped old caveats block, added shorter, more concise block about supporting only one Fortran compiler with Homebrew installs of json-fortran